### ISSUE TITLE
"Error: failed to render screenshot" and "Error: connect ETIMEDOUT" even though all tests pass

### ISSUE THREAD

**Issue by shlomitc** [applitools/eyes-storybook#12]
My eyes tests are running in CI but started to fail inconsistently.

I get `Error: connect ETIMEDOUT` and `Error: failed to render screenshot` messages, even though the tests are passing. It's either the `passing` indication is wrong, or something else is failing without any indication.

I wish I could provide you a reproducible example, but looks like it's network related issue, either on my side or applitools'.

any clue?

```bash
[23:13:13][npm run test] Using @applitools/eyes-storybook version 2.6.5.
[23:13:13][npm run test] 
[23:13:13][npm run test] - Starting storybook server
[23:13:28][npm run test] ✔ Storybook was started
[23:13:28][npm run test] - Reading stories
[23:13:30][npm run test] ✔ Reading stories
[23:13:30][npm run test] - Done 0 stories out of 49
[23:16:22][npm run test] ✖ Done 49 stories out of 49
[23:16:22][npm run test] 
[23:16:22][npm run test] [EYES: TEST RESULTS]:
[23:16:22][npm run test] Accordion: simple [1024x768] - Passed
[23:16:22][npm run test] AddItem/size: medium [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Announcement: Standard [1024x768] - Passed
[23:16:22][npm run test] Accordion: withButton [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Announcement: Footnote [1024x768] - Passed
[23:16:22][npm run test] AddItem/size: large [1024x768] - Passed
[23:16:22][npm run test] Accordion: multiple [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Announcement: DisabledAction [1024x768] - Passed
[23:16:22][npm run test] GooglePreview/basic: default render [1024x768] - Passed
[23:16:22][npm run test] Accordion: inCard [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Announcement: PremiumAction [1024x768] - Passed
[23:16:22][npm run test] GooglePreview/description: no description [1024x768] - Passed
[23:16:22][npm run test] AddItem/alignItems: center [1024x768] - Passed
[23:16:22][npm run test] MessageBox/AnnouncementRTL: Standard [1024x768] - Passed
[23:16:22][npm run test] GooglePreview/description: multiline [1024x768] - Passed
[23:16:22][npm run test] AddItem/alignItems: right [1024x768] - Passed
[23:16:22][npm run test] MessageBox/AnnouncementRTL: Footnote [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Alert: EmptyState [1024x768] - Passed
[23:16:22][npm run test] AddItem/alignItems: left [1024x768] - Passed
[23:16:22][npm run test] MessageBox/AnnouncementRTL: DisabledAction [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Alert: Standard [1024x768] - Passed
[23:16:22][npm run test] AddItem/theme: dashes [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Alert: Secondary [1024x768] - Passed
[23:16:22][npm run test] MessageBox/AnnouncementRTL: PremiumAction [1024x768] - Passed
[23:16:22][npm run test] RichTextInputArea/Editor: Bold [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Alert: Scrollable [1024x768] - Passed
[23:16:22][npm run test] RichTextInputArea/Editor: Italic [1024x768] - Passed
[23:16:22][npm run test] AddItem/theme: image [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Alert: Image [1024x768] - Passed
[23:16:22][npm run test] AddItem/disabled theme: dashes [1024x768] - Passed
[23:16:22][npm run test] MessageBox/Alert: FootNote [1024x768] - Passed
[23:16:22][npm run test] AddItem/disabled theme: plain [1024x768] - Passed
[23:16:22][npm run test] 
[23:16:22][npm run test] The following errors were found:
[23:16:22][npm run test] Error: failed to render screenshot d429d961-b58f-4df5-b185-364da2d3e113
[23:16:22][npm run test] Error: connect ETIMEDOUT 104.196.248.157:443
[23:16:22][npm run test] Error: failed to render screenshot d190fc32-f0de-4fd5-86e3-30bf40c3787f
[23:16:22][npm run test] Error: failed to render screenshot ba6f8333-0f99-4bb8-9f3e-2b60ef336887
[23:16:22][npm run test] Error: failed to render screenshot 647f9988-e4f8-4f5c-b202-0806e58acd6b
[23:16:22][npm run test] Error: failed to render screenshot 0cdb8b00-9da6-4199-b482-15017fbf2f5f
[23:16:22][npm run test] Error: failed to render screenshot 0c1ac45f-da35-4c23-9fd2-772bc92588ac
[23:16:22][npm run test] Error: failed to render screenshot 99d87f66-e02e-47c9-afb4-6e08dda0927c
[23:16:22][npm run test] Error: failed to render screenshot 5bf5afd3-0118-47c6-b8cb-cf843c9e98d1
[23:16:22][npm run test] Error: failed to render screenshot ccf474a5-295f-4e12-8116-65ad0b53088c
[23:16:22][npm run test] Error: failed to render screenshot afc017fa-f2b9-4b15-88c5-8416fd3c2a68
[23:16:22][npm run test] Error: failed to render screenshot d2f29f28-dc99-429e-abeb-b97e910f0829
[23:16:22][npm run test] Error: failed to render screenshot d58f5443-f3c8-40ee-908e-037d5948ef47
[23:16:22][npm run test] Error: failed to render screenshot 529a4dd8-e033-4fc5-9cf0-9bbb4897804f
[23:16:22][npm run test] Error: failed to render screenshot fcd6f704-7bd7-41d4-ba2a-ad869c8dc31f
[23:16:22][npm run test] Error: failed to render screenshot 6a04626a-073e-4751-a0ce-cfc23e768e13
[23:16:22][npm run test] Error: failed to render screenshot e74ef5e0-e37a-46a7-9128-a9dba42251ef
[23:16:22][npm run test] No differences were found!
[23:16:22][npm run test] 
[23:16:22][npm run test] See details at https://eyes.applitools.com/app/test-results/***********************
[23:16:22][npm run test] Total time: 174 seconds
[23:16:22][npm run test] 
```

**Comment by danielschwartz85** [comment]
Hi, thanks  for the issue - checking, will update

**Comment by danielschwartz85** [comment]
@shlomitc - can u add logs to your runs so we can get an error run log ?
By ```APPLITOOLS_SHOW_LOGS=true```  env variable or add ```showLogs=true``` to ```applitools.config.js```

**Comment by danielschwartz85** [comment]
@shlomitc - sorry for the late response. I think this issue may be solved in 3.6.7
(the timeout for rendering is now set to be higher than 2 minutes - makes sense that the rendering succeeded but later than 2 minutes - that's why the tests pass.)
Tnx

**Comment by shlomitc** [comment]
thanks. I'll close the issue and re-open if keeps happening
